scripts/lib/install

```diff
@@ -137,6 +138,7 @@ if [ "$has_appserver" = 0 ]; then
     ln -nsf "$deploy_path" /home/zulip/deployments/next
     ln -nsf "$deploy_path" /home/zulip/deployments/current
     ln -nsf /etc/zulip/settings.py "$deploy_path"/zproject/prod_settings.py
+    ln -nsf "$deploy_path"/zulip-thumbor-venv/bin/thumbor /usr/local/bin/thumbor
```

Comment:
I'm not sure this symlink makes sense to me; why not just have things using thumbor write `/home/zulip/deployments/current/zulip-thumbor-venv/bin/thumbor`?